views/partials/false_door.hbs

```diff
@@ -1,8 +1,8 @@
 <div id="false-door" class="hidden">
-  <div class='section-container'>
+  <div class="section-container">
     <h4>Thank you for trying Firefox Monitor</h4>
     <p>Firefox Monitor is a concept we are testing. During this test, we are not storing email addresses. This means that while we will use your email to give you real results about data breaches, we will not keep your email to alert you in case of future breaches</p>
     <p>We hope to provide this service soon, but in the meantime, you can stay up-to-date on Firefox Monitor and other new features when you sign up for the <a href='https://www.mozilla.org/newsletter/firefox/'>Firefox newsletter.</a></p>
```

Comment:
super minor nit, but not sure if you want to replace these `<a>` single quotes w/ doubles while you're at it.